android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUITableView.java

```diff
@@ -275,11 +275,11 @@ public void onDestroy(Activity activity)
 	@Override
 	public void release()
 	{
-		// Release search bar if there is one
+		// Release search bar views if there is one
 		if (nativeView instanceof RelativeLayout) {
 			((RelativeLayout) nativeView).removeAllViews();
 			TiViewProxy searchView = (TiViewProxy) (proxy.getProperty(TiC.PROPERTY_SEARCH));
-			searchView.release();
+			searchView.releaseViews();
```

Comment:
can searchView be null? Or is the layout instanced check an implicit way of "knowing" there's a search view?

Reply:
Yes, the layout check should only pass if we have a searchView and it is not part of the TableView (Its `searchAsChild` property is set to `false`).